### DISCUSSION TITLE
next.js examples : with-route-as-model is not working, problem in setting it up locally.

### DISCUSSION THREAD

**Discussion by AnchalDevBytes** [vercel/next.js#65835] [Help]
### Summary

Hey, I was trying to look into the example of how to create routing pages in a model, so wanted to see an example, here I found https://github.com/vercel/next.js/tree/canary/examples/with-route-as-modal , but when I tried to go create it locally with the given command, it throws an error, 
![image](https://github.com/vercel/next.js/assets/126651793/2fa5a2c5-8aca-42ac-9556-335876503cf7)

can anyone help me with it? or point to some resources where I can learn about how can I create routed pages inside a model popup, that I want to create on the screen.

**Comment by icyJoseph** [comment]
Hi,

The problem is that package managers are way more strict with peer dependency ranges now, and react-modal seems to allow only up to react@16, while we have gone past 17 and 18 already.

You can do `cd with-route-as-modal-app` and `npm i --force` to get it to work though. 

Though when I go to `react-modal`'s github page, they do hbave React 18 in their peer dependencies, so perhaps the problem is the react-modal version... 🤔

**Reply by icyJoseph** [reply]
@AnchalDevBytes a fix was merged, could you confirm that it works?

**Comment by AnchalDevBytes** [comment]
![image](https://github.com/vercel/next.js/assets/126651793/61f126a5-ba67-40f6-a6c2-19ef19feaed8)
Thanks @icyJoseph , its working

**Reply by icyJoseph** [reply]
Awesome, please don't forget to mark as answered. 👍
